gammapy/data/tests/test_observationlist.py

```diff
@@ -0,0 +1,33 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+from numpy.testing import assert_allclose
+from astropy.coordinates import SkyCoord
+from ...datasets import gammapy_extra
+from ...utils.testing import requires_data
+from ...utils.energy import EnergyBounds, Energy
+from ...data import DataStore, ObservationList
+
+
+@pytest.mark.xfail
+@requires_data('gammapy-extra')
+def test_make_psftable():
+    center = SkyCoord(83.63, 22.01, unit='deg')
+    store = gammapy_extra.filename("datasets/hess-crab4-hd-hap-prod2")
+    data_store = DataStore.from_dir(store)
+    obs1 = data_store.obs(23523)
+    obs2 = data_store.obs(23526)
+    energy = EnergyBounds.equal_log_spacing(1, 10, 100, "TeV")
+    energy_band = Energy([energy[0].value, energy[-1].value], energy.unit)
+    psf1 = obs1.make_psf(source_position=center, energy=energy, theta=None)
+    psf2 = obs2.make_psf(source_position=center, energy=energy, theta=None)
+    psf1_int = psf1.table_psf_in_energy_band(energy_band, spectral_index=2.3)
+    psf2_int = psf2.table_psf_in_energy_band(energy_band, spectral_index=2.3)
+    list = [obs1, obs2]
+    obslist = ObservationList(list)
+    psf_tot = obslist.make_psf(source_position=center, energy=energy)
+    psf_tot_int = psf_tot.table_psf_in_energy_band(energy_band, spectral_index=2.3)
+    assert_allclose(psf_tot_int.containment_radius(0.68), psf1_int.containment_radius(0.68), rtol=1e-3)
```

Comment:
After fixing the import, I get this test fail:
```
______________________________________________________________ test_make_psftable _______________________________________________________________

    @requires_data('gammapy-extra')
    def test_make_psftable():
        center = SkyCoord(83.63, 22.01, unit='deg')
        store = gammapy_extra.filename("datasets/hess-crab4-hd-hap-prod2")
        data_store = DataStore.from_dir(store)
        obs1 = data_store.obs(23523)
        obs2 = data_store.obs(23526)
        energy = EnergyBounds.equal_log_spacing(1, 10, 100, "TeV")
        energy_band = Energy([energy[0].value, energy[-1].value], energy.unit)
        psf1 = obs1.make_psf(source_position=center, energy=energy, theta=None)
        psf2 = obs2.make_psf(source_position=center, energy=energy, theta=None)
        psf1_int = psf1.table_psf_in_energy_band(energy_band, spectral_index=2.3)
        psf2_int = psf2.table_psf_in_energy_band(energy_band, spectral_index=2.3)
        list = [obs1, obs2]
        obslist = ObservationList(list)
        psf_tot = obslist.make_psf(source_position=center, energy=energy)
        psf_tot_int = psf_tot.table_psf_in_energy_band(energy_band, spectral_index=2.3)
>       assert_allclose(psf_tot_int.containment_radius(0.68), psf1_int.containment_radius(0.68), rtol=1e-3)
E       AssertionError: 
E       Not equal to tolerance rtol=0.001, atol=0
E       
E       (mismatch 100.0%)
E        x: Angle(<Angle 0.12588112016412742 deg>)
E        y: Angle(<Angle 0.11645282474146176 deg>)

gammapy/data/tests/test_observationlist.py:29: AssertionError
```
My recommendation would be that you change the test to this (with the imports at the top of the file):
```
    from astropy.coordinates import Angle
    from astropy.tests.helper import assert_quantity_allclose
    # Check that the mean PSF is consistent with the individual PSFs
    # (in this case the R68 of the mean PSF is in between the R68 of the individual PSFs)
    assert_quantity_allclose(psf1_int.containment_radius(0.68), Angle(0.11645282474146176, 'deg'))
    assert_quantity_allclose(psf2_int.containment_radius(0.68), Angle(0.14232666985400985, 'deg'))
    assert_quantity_allclose(psf_tot_int.containment_radius(0.68), Angle(0.12588112016412742, 'deg'))
```
I.e. put  assert_quantity_allclose statements against fixed values as regression tests, to establish the current results (so that we note when something changes or the method is improved in the future).

OK?